eo-maven-plugin/src/test/java/org/eolang/maven/HomeTest.java

```diff
@@ -105,4 +107,30 @@ void existsInDirWithSpecialSymbolsTest(@TempDir final Path temp) throws IOExcept
             Matchers.is(true)
         );
     }
+
+    @Test
+    void loadBytesFromExistingFile(@TempDir final Path temp) throws IOException {
+        final Home home = new Home();
+        final String filename = "foo";
+        final String content = "bar";
+        final Path subfolder = temp.resolve("subfolder").resolve(filename);
+        home.save(content, subfolder);
+        final Bytes bytes = home.load(subfolder);
+        final TextOf text = new TextOf(bytes);
+        MatcherAssert.assertThat(text, Matchers.equalTo(new TextOf(content)));
+    }
+
+    @Test
+    void loadFromAbsentFile(@TempDir final Path temp) {
```

Comment:
@volodya-lombrozo it's better to use `Assertions.assertThrows` from JUnit 5

Reply:
Done. BTW, can we use JUnit assertions instead of `MatcherAssert` in orher places?

Reply:
> Done. BTW, can we use JUnit assertions instead of `MatcherAssert` in orher places?

@volodya-lombrozo we use them somewhere, but JUnit assertions are way less powerful than Hamcrest.

Reply:
@volodya-lombrozo For consistency I'd use `MatcherAssert` everywhere for matching.

Reply:
@mximp  Even for exceptions?